src/Common/MemoryWorker.cpp

```diff
@@ -95,7 +95,7 @@ uint64_t readMetricsFromStatFile(ReadBufferFromFile & buf, std::initializer_list
         for (const auto * it = keys.begin(); it != keys.end(); ++it)
         {
             if (!(found_mask & (1l << (it - keys.begin()))))
-                LOG_ERROR(getLogger("CgroupsReader"), "Cannot find '{}' in '{}'", *it, buf.getFileName());
+                LOG_WARNING(LogFrequencyLimiter(getLogger("CgroupsReader"), 300), "Cannot find '{}' in '{}'", *it, buf.getFileName());
```

Comment:
I would remove those logs completely

Reply:
@al13n321 WDYT?

Reply:
Ok to remove, but I'd be paranoid that it may hide a bug. E.g. if there were a typo in `"kernel"`, we'd probably never notice and get slightly more OOMs because of that. Alternative: https://github.com/ClickHouse/ClickHouse/pull/77862